lib/Workflow/Manager.php

```diff
@@ -203,8 +203,17 @@ public function getWorkflowIfExists(object $subject, string $workflowName): ?Wor
         return $workflow;
     }
 
+    /**
+     * @deprecated will return ?WorkflowInterface in 12.0.0
+     */
     public function getWorkflowByName(string $workflowName): ?object
     {
+        trigger_deprecation(
+            'pimcore/pimcore',
+            '11.5.4',
+            'Method will return ?WorkflowInterface in 12.0.0'
+        );
```

Comment:
should we really add this deprecation message here? this will be fired all the time and developers cannot change anything to get rid of it?

Reply:
You are right, in this case, it is not good to trigger the message, because of no possible solution. I will remove it.